extensions/amp-list/0.1/amp-list.js

```diff
@@ -309,21 +364,157 @@ export class AmpList extends AMP.BaseElement {
     this.container_.dispatchEvent(event);
 
     // Change height if needed.
-    this.getVsync().measure(() => {
+    return this.getVsync().measurePromise(() => {
       const scrollHeight = this.container_./*OK*/scrollHeight;
       const height = this.element./*OK*/offsetHeight;
       if (scrollHeight > height) {
-        this.attemptChangeHeight(scrollHeight).catch(() => {});
+        return this.attemptChangeHeight(scrollHeight).catch(() => {});
+      } else {
+        Promise.resolve();
+      }
+    });
+  }
+
+  /**
+   * @private
+   */
+  setLoadMore_() {
+    if (!this.loadMoreSrc_ && !this.loadMoreOverflow_) {
+      return;
+    }
+
+    const triggerOnScroll = this.element.getAttribute('load-more') === 'auto';
+    if (triggerOnScroll) {
+      this.maybeSetupLoadMoreAuto_();
+    }
+
+    this.getVsync().mutate(() => {
+      this.loadMoreOverflow_ = this.getOverflowElement();
+      if (this.loadMoreOverflow_) {
+        this.loadMoreOverflow_.classList.toggle(
+            'amp-visible', !!this.loadMoreSrc_);
+        this.loadMoreOverflow_.onclick = () => this.loadMoreCallback_();
```

Comment:
` listen(elem, 'click', cb)` would be preferred (`listen` can be imported from `event-helper.js`)